molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -34,7 +34,7 @@
     </#if>
         <#if app_settings.customJavascript?has_content>
             <#list app_settings.customJavascript?split(r"\s*,\s*", "r") as js_file_name>
-                <script src="<@resource_href "${js_file_name?html}"/>"></script>
+                <script src="${js_file_name?html}"></script>
```

Comment:
The @resource_href macro adds a hash to the URL.
On a production server, we add a cache header for the js files that says they can be cached indefinitely.
But then we do need the hash or else they'll never be refreshed, even if they've changed.